devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy

```diff
@@ -182,7 +182,11 @@ def configure() {
 	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
 
 	log.debug "Configuring Reporting, IAS CIE, and Bindings."
-	def cmds = refresh() + zigbee.iasZoneConfig(30, 60 * 5) + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 60 * 30) + zigbee.enrollResponse()
+	def cmds = refresh() +
+			zigbee.configureReporting(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS, DataType.BITMAP16, 30, 60 * 5, null) +
+			zigbee.batteryConfig() +
+			zigbee.temperatureConfig(30, 60 * 30) +
+			zigbee.enrollResponse()
```

Comment:
should `enrollResponse()` be moved to the front here like Tom mentioned?

Reply:
`enrollResponse` is also called in `refresh` which is the first thing that happens here.  So previously we were sending 3 sets of `enrollResponse` commands, and I figured I'd drop it to 2

Reply:
👍